webapp/channels/src/packages/mattermost-redux/src/selectors/entities/channel_categories.ts

```diff
@@ -102,7 +102,7 @@ export function makeFilterAutoclosedDMs(): (state: GlobalState, channels: Channe
         getCurrentUserId,
         getMyChannelMemberships,
         getChannelMessageCounts,
-        getVisibleDmGmLimit,
+        (state) => getVisibleDmGmLimit(state),
```

Comment:
Interesting. I hadn't considered that adding an extra parameter to `getVisibleDmGmLimit` and such would make it so that other selectors which use it would also get an extra parameter by default. This is a good idea for fixing that though since I can think of a few other places where selectors parameters have leaked out in the past

Reply:
yeah, its an annoying side effect